Client/Frontend/Browser/HomePanel/FavoritesViewController.swift

```diff
@@ -296,7 +331,15 @@ class FavoritesViewController: UIViewController, Themeable {
         
         if !Preferences.NewTabPage.backgroundImages.value { return nil }
         
-        let isSponsoredImage = background?.sponsor != nil
+        let isSponsoredImage = { () -> Bool in
+            switch background?.backgroundType {
```

Comment:
If using a switch we should either implement each case explicitly, so we get compiler checking or just use:

```suggestion
            let isSponsoredImage = background?.backgroundType == .withBrandLogo
```

Reply:
Agreed, forgot to change it.

Reply:
this is fixed btw, just doesn't show as `outdated`